tests/backward-compat-hierarchical-ledger-manager/src/test/groovy/org/apache/bookkeeper/tests/TestCompatHierarchicalLedgerManager.groovy

```diff
@@ -0,0 +1,92 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements.  See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership.  The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package org.apache.bookkeeper.tests
```

Comment:
tests is a too broad package. I think it is better to have a sub package, like 'org.apache.bookkeeper.tests.backwardcompatiable` or `org.apache.bookkeeper.tests.bc`.

Reply:
why is the broadness a problem? We're not going to pull this into any other test to pollute the namespace. Using the broad namespace makes it easier to pull in utils, without having to declare each one.

Reply:
I am not saying it is a problem. what I am suggesting here is to use a `proper` package name to categorize those tests, rather than using a general tests. yes, we probably never pull those tests into others to pollute the namespace, but that is some sort of better practices to have `package` consideration in mind. 

> Using the broad namespace makes it easier to pull in utils, without having to declare each one.

not sure I am convinced about this. utils are kind of generic enough to be public to used across different namespaces.

Reply:
this is just a suggestion. feel free to ignore it. #approved to unblock your subsequent changes.

Reply:
I'll do it later. It's a minor change, but would be a pain across the whole patch series, so easier to do at the end. Added a note to #1030